src/components/profile/ProfileDirective.js

```diff
@@ -15,6 +15,7 @@ goog.require('ga_profile_service');
           templateUrl: 'components/profile/partials/profile.html',
           scope: {
             feature: '=gaProfile',
+            map: '=gaProfileMap',
```

Comment:
Not a big fan of bringing the map into this directive. Increases coupling.

I'm aware it was probably done because profile is now used for measure and draw and therefore the overlay code couldn't be handled in a single map-related directive anymore. But I really liked the decoupled profile directive. We can discuss it tomorrow.